data-serving/data-service/test/case.test.ts

```diff
@@ -1,5 +1,30 @@
 import request from 'supertest';
 import app from '../src/index';
+import mongoose from 'mongoose';
+import { Case } from '../src/model/case';
+
+beforeAll(async () => {
+    await mongoose.connect(
+        // This is provided by jest-mongodb.
+        // The `else testurl` is to appease Typescript.
+        process.env.MONGO_URL || 'testurl',
+        { useNewUrlParser: true, useCreateIndex: true },
```

Comment:
no useUnifiedTopology: true here? and why do we need useCreateIndex in tests?

Reply:
FWIW I din't need those in tests, I just used the same connection params as in index.ts

Reply:
SG, updated.